code/game/machinery/computer/camera_advanced.dm

```diff
@@ -76,7 +76,7 @@
 	playsound(src, 'sound/machines/terminal_off.ogg', 25, FALSE)
 
 /obj/machinery/computer/camera_advanced/check_eye(mob/user)
-	if( (stat & (NOPOWER|BROKEN)) || (!Adjacent(user) && !user.has_unlimited_silicon_privilege) || user.eye_blind || user.incapacitated() )
+	if( (stat & (NOPOWER|BROKEN)) || (!Adjacent(user) && !user.has_unlimited_silicon_privilege) ||user.incapacitated() ) //monkestation edit
```

Comment:
I do have to question **how** a camera console can be used by the blind.
Audio feedback?
Computer: "There is a Clown and a Captain in view. The clown is slipping the Captain. The Captain is malding."

Reply:
Yeah, what I pictured in my mind was effectively like that video I linked of a blind person browsing the internet.  Very fast text to speech that can also respond to inquiries.  Though talking purely game mechanics, blind people can use a number of different forms of camera or remote viewing, and enabling two more was easier and less disruptive than tracking down a ton more to disable!